test/tests/functional/pbs_provisioning_enhancement.py

```diff
@@ -331,7 +331,7 @@ def test_provisioning_with_placement(self):
         j.set_attributes({'Resource_List.select':
                           '1:ncpus=1:aoe=App1+1:ncpus=1',
                           'Resource_List.place': 'pack'})
-        j.set_sleep_time(5)
+        j.set_sleep_time(20)
```

Comment:
@Klovely Based on PR description we are not waiting for job to finish so I would say remove setting of sleep time as default sleep time is 100s